llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -798,9 +799,13 @@ define void @foo(i8 %val, ptr %ptr) {
   auto *BB = &*F->begin();
   auto It = BB->begin();
   auto *St = cast<sandboxir::StoreInst>(&*It++);
+  auto *VSt = cast<sandboxir::StoreInst>(&*It++);
   auto *Ret = cast<sandboxir::ReturnInst>(&*It++);
 
   // Check that the StoreInst has been created correctly.
+  // Check isVolatile()
```

Comment:
Remove the `// Check isVolatile()` comment.

Reply:
in a separate NFC PR, should i remove these comments off LoadInst tests too?

Reply:
Will do, thanks for the suggestion.